rust/azure_iot_operations_connector/Cargo.toml

```diff
@@ -17,8 +17,12 @@ publish = true
 # azure_iot_operations_mqtt = { version = "0.9", path = "../azure_iot_operations_mqtt", registry = "aio-sdks"  }
 # derive_builder.workspace = true
 # log.workspace = true
-# tokio.workspace = true
+tokio.workspace = true
 # tokio-util.workspace = true
+notify = "5.1"
+futures = "0.3"
```

Comment:
actually, are you using tokio-stream or futures at this point? If you know you'll use them for the implementation, feel free to leave them in the Cargo.toml for now, but just comment them out.